examples/lkj.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import torch
+
+import pyro
+import pyro.distributions as dist
+from pyro.infer.mcmc import MCMC, NUTS
+
+"""
+This simple example is intended to demonstrate how to use an LKJ prior with
+a multivariate distribution.
+
+It generates entirely random, uncorrelated data, and then attempts to fit a correlation matrix
+and vector of variances.
+"""
+
+
+def model(y):
+    d = y.shape[1]
+    N = y.shape[0]
+    # Vector of variances for each of the d variables
+    theta = pyro.sample("theta", dist.HalfCauchy(torch.full((d,), 1, dtype=y.dtype)))
+    # Lower cholesky factor of a correlation matrix
+    eta = y.new_full((1,), 1)
+    L_omega = pyro.sample("L_omega", dist.CorrLCholeskyLKJPrior(d, eta))
+    # Lower cholesky factor of the covariance matrix
+    L_Omega = torch.mm(torch.diag(theta.sqrt()), L_omega)
+
+    # Vector of expectations
+    mu = y.new_zeros(d)
+
+    with pyro.plate("observations", N) as n:
+        obs = pyro.sample("obs", dist.MultivariateNormal(mu, scale_tril=L_Omega), obs=y[n])
+    return obs
+
+
+def main(args):
+    y = torch.randn(args.n, args.num_variables).to(dtype=torch.double)
+    nuts_kernel = NUTS(model, jit_compile=args.jit, step_size=1e-5)
+    MCMC(nuts_kernel, num_samples=args.num_samples, warmup_steps=args.warmup_steps, num_chains=args.num_chains).run(y)
+
+
+if __name__ == "__main__":
+    assert pyro.__version__.startswith('0.3.0')
```

Comment:
We just had a 0.3.1 release yesterday, so this should be `0.3.1` otherwise it will throw an error.